pkg/handlers/arhandler.go

```diff
@@ -0,0 +1,63 @@
+package handlers
```

Comment:
Think the naming should be `AR` since it is already part of the `handlers` package.